docs/examples/resnet50_trt/client.py

```diff
@@ -95,4 +95,4 @@ def load_image(img_path: str):
 
     maxs = np.argmax(output0_data, axis=1)
 
-    print("{}ms class: {}".format(latency, labels_dict[maxs[0]]))
+    print("{}ms class: {}".format(latency * 1000, labels_dict[maxs[0]]))
```

Comment:
Hi @laochonlam , thank you for the contribution!

While you are at it, could you change this legacy formatting way to the better one?
```suggestion
    print(f"{latency * 1000}ms class: {labels_dict[maxs[0]]}")
```

Reply:
Sure!